lms/static/sass/views/_login-register.scss

```diff
@@ -350,6 +350,8 @@ $sm-btn-linkedin: #0077b5;
         @extend %btn-secondary-grey-outline;
         width: 130px;
         padding: 0 0 0 ($baseline*2);
+        margin-right: 6px;
+        margin-bottom: $baseline;
         height: 34px;
```

Comment:
This is also not something I changed. I can convert this to `min-height` if you want, but then you get this (sorry this is in RTL mode, but it still illustrated the point):

![screen shot 2015-06-25 at 4 11 12 pm](https://cloud.githubusercontent.com/assets/945577/8367608/ecaff7ba-1b54-11e5-9620-aa2d18ca0dbe.png)

Currently the buttons don't expand at all, and extra text gets cut off. An alternative is to change the `width` to `min-width`, which I think is less bad:
![screen shot 2015-06-25 at 4 13 08 pm](https://cloud.githubusercontent.com/assets/945577/8367630/2cec216e-1b55-11e5-8ecc-6ca1b84ef42b.png)

That said, this styles are not part of my change so let me know if we can postpone this.